### PR TITLE
doc: update README.md

### LINKED ISSUE #1761: doc: Chinese doc
English version: https://github.com/apache/incubator-apisix/pull/1757/files

we need to update the Chinese doc too.

### PR DESCRIPTION
**Pull request by scaat**
NOTE: Please read the Contributing.md guidelines before submitting your patch:

https://github.com/apache/incubator-apisix/blob/master/Contributing.md#how-to-add-a-new-feature-or-change-an-existing-one

### Summary

SUMMARY_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #1761

### CODE REVIEW

File: doc/zh-cn/README.md [hunk 1]
```diff
@@ -69,3 +69,17 @@
...
+
+### AWS
+
+推荐的方法是在 [AWS Fargate](https://aws.amazon.com/fargate/) 上使用  [AWS CDK](https://aws.amazon.com/cdk/) 部署 APISIX，这有助于将 APISIX 层和上游层分离到具有自动缩放功能的完全托管和安全的无服务器容器计算环境之上。
+
+请参阅 [Pahud Hsieh](https://github.com/pahud) 撰写的[指南](https://github.com/pahud/cdk-samples/blob/master/typescript/apisix/README.md) ，了解如何在 AWS CDK 中 100％ 配置推荐的架构。
+
+### Kubernetes
+
+请参阅[指南](../kubernetes/README.md)并了解如何在 Kubernetes 中部署 apisix。
```
Comment (membphis) [inline doc/zh-cn/README.md:84]: `../kubernetes/README.md` is wrong, please fix it

Comment (membphis) [reply doc/zh-cn/README.md:84]: Is `/kubernetes/README.md` correct?

Comment (scaat) [reply doc/zh-cn/README.md]: fix

File: doc/zh-cn/README.md [hunk 2]
```diff
@@ -69,3 +69,17 @@
...
 * [oauth](plugins/oauth.md): 提供 OAuth 2 身份验证和自省。
+
+部署
+=======
+
+### AWS
+
+推荐的方法是在 [AWS Fargate](https://aws.amazon.com/fargate/) 上使用  [AWS CDK](https://aws.amazon.com/cdk/) 部署 APISIX，这有助于将 APISIX 层和上游层分离到具有自动缩放功能的完全托管和安全的无服务器容器计算环境之上。
+
+请参阅 [Pahud Hsieh](https://github.com/pahud) 撰写的[指南](https://github.com/pahud/cdk-samples/blob/master/typescript/apisix/README.md) ，了解如何在 AWS CDK 中 100％ 配置推荐的架构。
```
Comment (juzhiyuan) [inline doc/zh-cn/README.md:80]: `README.md) ，了解` it seems that there has 1 extra space before `，`

File: doc/zh-cn/README.md [hunk 3]
```diff
@@ -69,3 +69,17 @@
 * [authz-keycloak](plugins/authz-keycloak-cn.md): 支持 Keycloak 身份认证服务器
 * [uri-blocker](plugins/uri-blocker.md): 根据 URI 拦截用户请求。
 * [oauth](plugins/oauth.md): 提供 OAuth 2 身份验证和自省。
+
+部署
```
Comment (moonming) [inline doc/zh-cn/README.md:73]: better also add the English verison

File: doc/zh-cn/README.md [hunk 4]
```diff
@@ -69,3 +69,17 @@
...
+
+### AWS
+
+推荐的方法是在 [AWS Fargate](https://aws.amazon.com/fargate/) 上使用  [AWS CDK](https://aws.amazon.com/cdk/) 部署 APISIX，这有助于将 APISIX 层和上游层分离到具有自动缩放功能的完全托管和安全的无服务器容器计算环境之上。
+
+请参阅 [Pahud Hsieh](https://github.com/pahud) 撰写的[指南](https://github.com/pahud/cdk-samples/blob/master/typescript/apisix/README.md)，了解如何在 AWS CDK 中 100％ 配置推荐的架构。
+
+### Kubernetes
+
+请参阅[指南](../../kubernetes/README.md)并了解如何在 Kubernetes 中部署 apisix。
```
Comment (moonming) [inline doc/zh-cn/README.md:84]: APISIX, uppercase letters

Comment (moonming) [inline doc/zh-cn/README.md:84]: @gxthrj Is Kubernetes doc ready?

File: doc/zh-cn/README.md [hunk 5]
```diff
@@ -69,3 +69,17 @@
 * [authz-keycloak](plugins/authz-keycloak-cn.md): 支持 Keycloak 身份认证服务器
 * [uri-blocker](plugins/uri-blocker.md): 根据 URI 拦截用户请求。
 * [oauth](plugins/oauth.md): 提供 OAuth 2 身份验证和自省。
+
+部署
+=======
+
+### AWS
+
+推荐的方法是在 [AWS Fargate](https://aws.amazon.com/fargate/) 上使用  [AWS CDK](https://aws.amazon.com/cdk/) 部署 APISIX，这有助于将 APISIX 层和上游层分离到具有自动缩放功能的完全托管和安全的无服务器容器计算环境之上。
```
Comment (moonming) [inline doc/zh-cn/README.md:78]: Is this document available for Apache APISIX 1.3? This doc is about 8 months ago.

File: doc/zh-cn/README.md (hunk 3 above)
Comment (scaat) [reply doc/zh-cn/README.md:73]: English version already available in https://github.com/apache/incubator-apisix/blob/master/doc/README.md

File: doc/zh-cn/README.md (hunk 4 above)
Comment (scaat) [reply doc/zh-cn/README.md]: fix

**Comment by gxthrj** [comment]
I will make a try later according to the doc.

**Pull request merged**